homeassistant/components/media_player/sonos.py

```diff
@@ -388,6 +388,18 @@ def name(self):
         """Return the name of the device."""
         return self._name
 
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return {
+            'identifiers': [
+                (SONOS_DOMAIN, self._unique_id)
+            ],
+            'name': self._name,
+            'model': self._model.replace("Sonos ", ""),
```

Comment:
There was a recent firmware change so "Sonos " might not be right for all players (cf. #15604).

On the other hand, Sonos is pretty agressive about updates so maybe it no longer matters.

Reply:
If the string is not found, it does nothing 👍